tests/ident/test_engine_parser_xtandem_alanine.py

```diff
@@ -156,15 +150,14 @@ def test_engine_parsers_xtandem_unify_row():
             "15N": False,
         },
     )
-    for row in parser:
-        assert (
-            row["Raw data location"]
-            == "/Users/cellzome/Dev/Gits/Ursgal/ursgal2_dev/tests/data/test_Creinhardtii_QE_pH11.mgf"
-        )
-        assert row["Sequence"] == "DDVHNMGADGIR"
-        assert row["Modifications"] == "Oxidation:6"
-        assert row["Search Engine"] == "xtandem_alanine"
-        break
+    first_row = parser.unify().iloc[0, :]
+    assert (
+        first_row["Raw data location"]
+        == "/Users/cellzome/Dev/Gits/Ursgal/ursgal2_dev/tests/data/test_Creinhardtii_QE_pH11.mzML"
+    )
+    assert first_row["Sequence"] == "DDVHNMGADGIR"
+    assert first_row["Modifications"] == "Oxidation:6"
+    assert first_row["Search Engine"] == "xtandem_alanine"
```

Comment:
same as above - please also assert identity of the resulting df - not just first line :)